src/Context.ts

```diff
@@ -31,23 +31,23 @@ export function CreateContextAndInputs(info: FunctionInfo, request: rpc.IInvocat
     context.res = new Response(context.done);
   }
   return {
-    context: <IContext>context,
+    context: <Context>context,
     inputs: inputs
   }
 }
 
-class Context implements IContext {
+class InvocationContext implements Context {
   invocationId: string;
-  executionContext: IExecutionContext;
-  bindings: IDict<any>;
-  bindingData: IDict<any>;
-  bindingDefinitions: IBindingDefinition[];
-  log: ILogger;
+  executionContext: ExecutionContext;
+  bindings: Dict<any>;

```

Comment:
I wouldn't recommend introducing a generic Dict interface for just Functions.

Either do an anonymous type or do a type specific to each property. 

Also, the bindings can't contain "any". There is definitely a subset of types they support. I don't recommend shipping "any" on public types.

Reply:
Agreed. Since we're not publishing the types from Context.ts though, I think I'll file an issue to remove `any` and save it for later!